### PR TITLE
ZKUI-86 : add error message on bucket creation

### PR DESCRIPTION
**Pull request by gaudiauj**
add error message and ui validation on bucket creation : `Bucket names can include only lowercase letters, numbers, dots (.), and hyphens (-)`

### CODE REVIEW

File: src/react/databrowser/buckets/BucketCreate.jsx [hunk 1]
```diff
@@ -174,8 +178,9 @@ function BucketCreate() {
                 tooltipMessages={[
                   'Must be unique',
                   'Cannot be modified after creation',
+                  bucketErrorMessage,
```
Comment (ChengYanJin) [inline src/react/databrowser/buckets/BucketCreate.jsx:181]: I think we were saying we don't need to add this information to the tooltip, only the dynamic validation error is enough.
Can you double-check with @Cuervino ?

Comment (gaudiauj) [reply src/react/databrowser/buckets/BucketCreate.jsx:181]: he commented on the ticket : `This text will also be added in the tooltip next to “Bucket name”.`
but i'll ask him anyway

Comment (gaudiauj) [reply src/react/databrowser/buckets/BucketCreate.jsx:181]: he confirmed this has to go on the tooltip as well :)

**Review by ChengYanJin — APPROVED** [review]
LGTM!

File: src/react/databrowser/buckets/BucketCreate.jsx (hunk 1 above)
Comment (ChengYanJin) [reply src/react/databrowser/buckets/BucketCreate.jsx:181]: OK. sorry! I didn't really check the comments on the tickets. Thanks @Cuervino !

**Review by Cuervino — APPROVED** [review]
LGTM

**Pull request merged**